src/blocks/author-list/view.js

```
@@ -1,80 +1,84 @@
-            // Open in new window
-            link.setAttribute('target', '_blank');
-            link.setAttribute('rel', 'noopener noreferrer');
+	// Initialize social link interactions
+	const socialLinks = block.querySelectorAll('.apbl-social-item a');
+	if (socialLinks.length) {
+		socialLinks.forEach((link) => {
+			// Open in new window
+			link.setAttribute('target', '_blank');
+			link.setAttribute('rel', 'noopener noreferrer');
+
+			link.addEventListener('click', (e) => {
```

ESLint

[error] 'e' is defined but never used.

(no-unused-vars)